CHANGELOG.md

```diff
@@ -1,26 +1,10 @@
 # Change Log
 
-## Version 0.1.8
+## Unreleased: Version 0.1.8
```

Comment:
?

Reply:
I made the mistake of adding upcoming changelog items without marking it as unreleased in a previous PR.

Reply:
Removing the version from here